bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -481,21 +482,27 @@ class PresentationUploader extends Component {
   }
 
   render() {
-    const { intl } = this.props;
+    const { intl, mountModal } = this.props;
     const { preventClosing, disableActions } = this.state;
 
     return (
       <ModalFullscreen
         title={intl.formatMessage(intlMessages.title)}
         preventClosing={preventClosing}
         confirm={{
-          callback: this.handleConfirm,
+          callback: () => {
+            mountModal(null);
```

Comment:
the modal should close only when all presentations are ready. do not have how to post a message when it starts uploading and when it's ready, a toast notification would not solve it?